cli/src/klio_cli/commands/message/publish.py

```diff
@@ -89,7 +89,9 @@ def _publish_messages(
             message = bytes(entity_id.encode("utf-8"))
 
         try:
-            publish(data=message)
+            future = publish(data=message)
+            # block until message is published or exception is raised
+            future.result()
```

Comment:
Just double checking: I know this was related to a library update but is `future.result()` backwards compatible?

Reply:
Yep, since we already require `google-cloud-pubsub>=2.0.0` and I tested successfully with `2.0.0`